src/luax.c

```diff
@@ -215,6 +215,28 @@ int luax_getstack(lua_State* L) {
   return 1;
 }
 
+// Variant of luax_getstack for panic (doesn't assume free memory, skips nothing in traceback)
+int luax_getstack_panic(lua_State *L) {
+  if (!lua_checkstack (L, 3)) // Maybe this could be 2, depends on how lua_insert works
+    return 0;
+  lua_getglobal(L, "debug");
+  if (!lua_istable(L, -1)) {
+    lua_pop(L, 1);
+    return 0;
+  }
+  lua_getfield(L, -1, "traceback");
+  if (!lua_isfunction(L, -1)) {
+    lua_pop(L, 2);
+    return 0;
+  }
+  lua_remove (L, -2);     // Pop debug object
+  lua_insert(L, -2);      // Move message to top
+  lua_pushinteger(L, 0);
+  lua_call(L, 2, 1);      // Call debug.traceback
+  return 1;
+}
```

Comment:
Fixed the various style changes. Your suggestion of merging the two getstacks seems reasonable but I think you have something kinda specific in mind so I'd prefer to let you handle that after merge.

Since we only have two levels we want (2 or 0) another way to do this without relying on closures would be to have one `lua_getstack(lua_State *L, int errorLevel)` and then have two hardcoded `lua_getstack_2` and `lua_getstack_0` C functions.